DetectDynamicJS.py

```diff
@@ -77,6 +77,11 @@ def doPassiveScan(self, baseRequestResponse):
         statusCode = responseInfo.getStatusCode()
         if statusCode != 200:
           return
+        
+        # Check if the script is world readable
```

Comment:
Same here :)
Found them while doing a pentest, in real environment. Jus wanted to update things on your end.